frontend/src/app/features/work-packages/components/wp-baseline/baseline/baseline.component.sass

```diff
@@ -51,7 +51,7 @@
       border: 1px solid $spot-color-basic-gray-4
       border-left: none !important
       padding: $spot-spacing-0_5 $spot-spacing-0_25
-      line-height: unset
+      line-height: 1rem
```

Comment:
This change results in the time info being not correclty aligned

<img width="113" alt="Bildschirmfoto 2023-08-03 um 09 26 58" src="https://github.com/opf/openproject/assets/7457313/518a4450-e7f8-4ca0-81d1-287ca321ccc0">